vkubelet/pod.go

```diff
@@ -125,6 +140,10 @@ func (s *Server) syncPod(ctx context.Context, pod *corev1.Pod) {
 			logger.WithError(err).Error("Failed to delete pod")
 		}
 	} else {
+		if stored := s.resourceManager.GetPod(pod.GetName(), pod.GetNamespace()); stored != nil {
```

Comment:
this check is not correct, as we add the pod into the resource manager before adding it into the podSync channel.

Reply:
oh my this is so wrong. It's not preventing things from working because the call is incorrect!